shell/config/router/navigation-guards/attempt-first-login.js

```diff
@@ -5,10 +5,10 @@ import { tryInitialSetup } from '@shell/utils/auth';
 import { routeNameMatched } from '@shell/utils/router';
 
 export function install(router, context) {
-  router.beforeEach((from, to, next) => attemptFirstLogin(from, to, next, context));
+  router.beforeEach((to, from, next) => attemptFirstLogin(to, from, next, context));
```

Comment:
That's a good catch, were `to` and `from` always inverted from the initial implementation?

Reply:
It was.